K2Field.K2NE.ServiceBroker/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.IO.Compression" version="4.3.0" targetFramework="net45" />
```

Comment:
I'm not sure this package will be available on the server, which runs .NET Framework 4.8.
If this causes issues on deployment, consider using [GZipStream](https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.gzipstream?view=netframework-4.8), that should be available on the server.System.dll.